ergo/actions/clean.py

```diff
@@ -3,19 +3,11 @@
 
 from ergo.project import Project
 
-def usage():
-    print("usage: ergo clean <path> [args]")
-    quit()
-
-def parse_args(argv):
-    parser = argparse.ArgumentParser(description="cleaner")
+def clean_args(subparsers, name, desc):
```

Comment:
these xxx_args() functions injects a subparser with the current argument into the main parser